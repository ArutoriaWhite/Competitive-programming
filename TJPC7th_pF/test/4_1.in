6206519 983
4017 1
17312 0
19125 0
21993 0
23339 1
24533 1
27450 1
40602 1
52843 0
55194 1
56781 0
59754 0
60276 1
61676 0
77500 1
77837 0
77993 1
82834 1
92640 1
120718 0
124075 0
124599 1
133986 0
134588 0
139499 0
149356 1
150556 0
152389 0
152997 1
165136 0
169657 0
171463 0
179026 1
179588 1
187647 1
200070 0
219510 1
223109 1
224306 0
228958 1
236001 0
256774 0
265513 1
271890 0
286377 0
289537 0
301974 1
304686 1
313419 0
316228 1
319091 0
326852 1
330936 0
332765 1
333139 0
340839 1
345822 0
358823 1
362619 1
375308 0
385767 0
396806 1
397265 1
403950 0
405252 0
405716 0
412503 1
418024 0
421718 0
423502 1
427262 0
428336 1
439703 1
440607 1
442459 0
445903 0
457142 1
459449 1
464417 0
466494 0
469344 0
474607 1
478403 0
486309 0
491022 1
493801 1
494609 1
498473 0
500594 0
501546 1
505641 1
510481 0
525886 0
527119 1
534082 0
534424 1
543740 0
550569 1
564579 1
564865 0
588070 1
593657 0
593751 0
598808 1
601524 0
602767 1
603721 1
612745 0
615623 1
621774 0
622376 1
634263 1
635584 0
637442 1
642984 0
645057 0
646712 0
649115 1
653657 1
658321 0
669773 1
673501 0
689417 1
707357 1
707403 1
707993 1
711930 0
716086 1
727033 0
732581 0
738458 1
740954 1
746143 0
746696 0
747067 1
750280 0
756205 1
758226 0
765685 0
774659 1
776445 1
780907 0
794565 1
802058 1
820372 1
820676 1
822889 1
823387 1
838026 1
849666 0
859763 1
865527 0
868841 1
870417 1
878457 0
885113 0
902256 0
904973 1
911904 0
916007 1
918436 0
918518 1
923172 1
923757 0
928389 1
930408 0
932503 0
933337 0
935016 1
937964 0
947166 0
954658 1
956225 0
956948 0
957388 0
975006 0
978153 0
982762 1
984976 0
991080 0
993026 0
1002714 1
1024718 0
1027140 0
1029554 1
1030150 0
1039960 0
1041416 1
1044861 0
1049655 0
1052772 1
1058262 1
1083788 1
1086569 0
1088547 1
1103872 1
1107484 0
1108169 0
1117788 0
1120693 1
1124004 0
1124862 1
1127367 1
1136287 1
1141664 1
1145432 0
1155489 1
1177327 1
1204660 1
1208221 1
1215681 1
1216828 0
1220381 0
1225150 1
1228856 0
1229634 0
1231028 1
1233046 0
1239277 0
1250406 0
1260064 1
1260331 1
1266242 1
1274135 1
1276267 1
1278765 0
1280239 0
1293346 0
1304135 0
1309949 1
1319467 0
1330119 0
1333615 1
1343291 1
1352965 0
1364551 0
1367187 0
1370354 0
1379805 1
1380187 0
1380900 1
1383490 1
1389712 0
1390195 1
1400620 0
1412847 0
1416473 1
1428608 0
1432544 1
1445718 1
1456159 1
1469748 1
1472552 1
1473065 1
1475015 0
1476856 1
1495871 1
1503961 0
1508759 0
1526097 0
1526309 1
1527091 0
1535191 0
1546301 0
1560550 0
1565517 1
1566655 0
1568084 0
1572413 0
1575024 0
1584484 0
1591222 0
1600475 0
1609770 0
1615155 1
1618555 1
1623062 0
1624793 1
1627558 1
1628508 0
1628536 1
1636422 0
1642788 0
1660229 1
1673008 1
1683637 0
1687885 1
1691203 0
1692967 0
1702130 1
1703740 1
1724794 1
1725255 0
1731084 1
1731498 1
1745996 1
1749142 1
1749422 1
1759407 0
1760019 1
1760200 0
1770209 0
1775077 1
1779275 0
1787945 1
1789514 1
1801697 1
1802849 1
1804750 0
1805427 0
1806089 1
1809296 0
1813329 1
1829312 1
1831978 1
1842834 1
1865404 0
1877737 0
1884504 0
1889485 0
1898499 0
1903771 1
1912460 0
1915107 1
1915293 1
1920962 1
1921488 1
1925143 0
1925409 1
1926962 1
1929650 0
1932602 0
1938367 1
1940454 0
1945352 0
1950540 0
1953952 1
1961315 0
1966559 1
1988468 0
1995844 1
1999306 0
2000282 0
2000851 0
2006034 0
2016463 1
2018350 0
2021330 1
2037568 0
2048943 1
2066036 1
2068966 1
2074484 1
2079142 0
2093649 0
2101461 1
2104419 1
2110202 0
2116564 0
2122914 1
2128275 1
2131167 0
2132576 0
2134408 0
2134869 1
2151333 1
2171429 0
2174481 1
2182364 0
2183258 0
2183437 1
2187316 0
2191805 0
2207199 1
2229771 1
2232751 1
2245370 1
2252474 0
2258314 1
2271072 0
2275828 1
2281021 1
2291683 0
2301163 1
2308226 0
2314837 0
2319297 0
2327320 1
2339564 1
2339627 0
2349754 0
2351382 0
2352635 0
2356129 0
2380956 1
2391236 0
2392116 1
2406758 0
2416780 0
2420663 1
2422475 0
2424691 0
2425272 1
2432935 0
2436486 0
2437184 1
2449873 0
2457382 1
2457477 0
2471445 0
2480855 0
2486515 1
2486859 0
2493415 1
2497264 0
2498405 1
2498869 0
2508429 0
2515827 0
2523553 0
2524462 0
2536018 0
2546899 0
2552062 0
2552083 0
2563676 0
2564228 1
2577900 0
2580526 1
2582657 1
2600062 1
2600437 1
2628163 1
2638754 0
2643423 1
2653645 0
2662055 0
2662715 1
2666228 0
2670180 0
2674961 0
2688235 0
2698668 0
2711551 1
2728967 1
2758336 1
2763547 0
2766505 1
2776564 0
2788239 1
2805868 0
2811039 1
2825960 0
2830450 0
2831944 0
2848043 1
2850674 0
2864780 1
2871076 0
2894063 0
2903142 0
2905528 1
2918220 0
2931309 1
2939239 1
2939651 1
2941001 1
2941337 0
2942403 0
2958905 1
2970907 1
2977149 1
2980836 1
2984144 0
2985900 0
2993921 0
2996998 1
3008429 0
3026048 0
3028347 0
3042236 1
3048398 0
3054621 1
3058706 0
3059576 0
3061531 1
3064629 1
3072943 0
3081206 1
3092759 1
3097221 1
3098952 0
3105858 0
3111416 0
3119038 1
3127572 0
3135372 0
3135631 1
3141883 1
3161593 0
3162777 1
3164303 0
3165102 1
3166742 0
3168802 1
3169256 1
3173930 0
3190518 0
3193474 0
3194385 1
3200877 0
3208380 1
3208627 0
3213744 1
3214158 1
3227845 0
3234281 1
3238408 1
3241272 0
3248819 0
3250030 0
3252619 0
3256879 0
3259597 0
3262881 0
3268399 0
3273565 0
3274201 1
3293076 0
3293493 0
3305504 0
3316786 1
3319602 1
3334435 1
3345677 0
3347296 1
3348804 0
3352411 0
3358261 1
3362878 1
3365221 0
3378304 1
3382950 1
3388669 1
3393780 0
3400688 1
3404168 0
3405427 1
3408325 1
3413226 0
3419723 0
3420296 1
3424452 0
3424989 1
3425929 1
3433714 1
3440838 1
3441296 0
3444609 0
3446551 0
3449748 0
3455971 1
3459293 1
3473475 1
3494329 0
3495364 0
3497030 1
3511995 0
3525007 0
3532724 1
3533970 1
3540539 1
3544011 1
3547371 1
3556034 0
3560244 0
3571720 1
3577246 1
3579222 0
3593822 0
3606413 0
3613901 0
3614956 1
3619307 0
3620006 0
3630541 1
3647003 1
3648327 0
3662220 0
3663688 0
3669587 0
3671081 1
3676026 0
3678598 0
3681939 1
3683686 1
3691372 0
3694376 0
3698635 0
3703387 0
3711481 0
3714925 1
3717682 1
3719560 0
3725628 1
3732744 1
3737220 0
3741901 0
3746742 1
3761338 1
3763982 0
3765560 1
3766513 0
3772630 0
3776163 0
3781147 1
3785159 1
3803812 1
3818231 1
3821798 0
3843896 0
3846600 1
3847299 1
3848614 1
3855074 0
3872647 1
3873725 0
3876099 0
3878253 0
3885927 1
3895342 0
3898007 1
3898182 0
3913002 1
3917559 1
3922705 0
3928975 0
3937330 0
3947127 1
3948180 0
3949081 0
3950173 1
3955008 0
3958126 1
3962861 1
3975023 0
3977846 0
3985825 1
3993688 1
3994096 0
3996268 0
3998366 1
4000402 1
4010647 0
4023662 0
4027646 0
4031127 1
4038739 1
4058576 0
4064316 0
4067535 0
4069715 1
4071721 1
4079913 1
4079961 1
4089629 1
4090622 1
4096313 1
4109026 1
4114290 0
4130547 0
4141541 0
4145903 0
4147535 0
4161315 1
4165238 1
4181708 1
4186445 0
4188677 0
4191147 0
4196724 0
4224419 1
4231104 0
4233397 1
4234999 1
4238964 0
4245573 1
4245732 0
4246680 1
4252286 0
4253482 0
4256506 0
4258126 1
4282803 0
4285143 0
4287075 0
4294434 0
4303008 0
4316634 0
4318637 0
4324505 0
4332858 1
4333018 0
4334494 0
4336845 0
4346822 1
4357132 1
4357209 1
4379641 1
4381847 0
4393124 1
4396401 0
4398415 1
4401895 1
4402904 0
4422498 0
4423934 1
4430435 0
4430787 0
4441177 1
4454558 0
4455164 1
4455244 1
4458599 1
4463806 1
4468715 0
4478525 1
4479964 0
4488157 1
4511451 0
4511591 1
4532971 0
4544361 0
4550327 1
4550529 0
4576659 0
4581893 1
4585204 0
4586502 0
4593151 1
4595315 1
4603663 0
4606701 0
4608701 1
4611259 1
4642410 0
4645044 1
4645323 1
4648063 0
4658915 0
4678994 0
4679221 0
4682106 0
4683871 0
4691995 0
4693411 0
4727088 1
4727586 0
4728140 0
4738613 0
4743725 1
4755077 0
4756222 0
4764373 0
4786780 1
4789155 0
4792832 0
4803281 1
4811757 0
4819308 0
4821982 1
4826392 1
4835184 0
4840148 1
4844355 0
4846963 0
4851968 1
4863849 1
4872331 1
4878413 0
4883685 0
4902222 0
4904027 1
4907956 0
4908343 1
4908543 1
4912967 0
4920989 1
4930943 0
4932744 1
4934038 0
4941505 0
4948955 0
4949683 1
4949946 1
4951543 1
4954655 0
4961320 1
4965555 1
4985908 0
4990256 0
5004120 0
5004299 1
5017778 1
5020702 1
5020844 1
5029718 1
5042569 0
5056983 0
5063676 1
5066268 1
5066910 1
5068795 1
5070369 0
5071308 1
5075338 1
5078187 1
5102774 1
5103162 1
5105566 0
5107721 1
5110351 1
5115236 0
5115783 1
5121802 0
5125442 0
5125953 0
5126466 0
5130538 1
5132538 0
5150038 0
5150981 0
5153395 1
5168621 0
5169463 1
5173781 1
5185877 0
5188570 1
5198694 1
5200554 1
5206416 0
5208868 0
5209032 0
5210719 1
5215027 1
5224585 0
5234139 0
5235747 1
5236848 1
5239432 1
5247041 0
5248254 0
5258533 1
5273551 0
5278750 1
5286218 1
5287344 0
5298620 1
5306022 1
5306583 1
5306898 0
5310496 0
5314497 1
5316758 0
5320042 1
5320238 0
5321939 1
5324033 1
5328658 1
5353236 1
5355778 1
5366669 1
5370360 0
5370575 1
5370835 1
5379614 1
5384411 1
5390617 1
5392558 0
5395695 0
5411039 0
5446424 1
5453409 1
5456048 0
5473059 1
5483301 1
5485330 0
5505273 0
5509488 1
5512162 0
5522766 1
5528097 0
5531211 0
5533913 1
5551489 0
5553242 0
5575710 0
5581003 0
5583147 1
5589854 1
5600126 1
5600309 0
5617831 0
5618455 0
5619449 0
5620909 0
5628463 1
5631938 1
5635683 0
5655091 0
5655223 1
5664585 0
5679031 1
5690401 1
5693039 1
5706957 1
5709287 1
5718707 1
5729528 1
5729535 1
5742018 0
5751670 0
5752263 1
5757322 0
5759612 1
5761853 1
5767464 1
5795020 1
5795901 1
5803718 0
5835079 1
5847268 0
5860089 0
5861571 1
5862755 0
5877165 0
5879168 0
5880351 0
5884301 0
5888006 0
5888928 0
5912502 0
5913914 0
5917664 1
5919834 1
5920945 1
5947614 0
5949136 1
5951224 0
5958151 1
5968654 1
5973815 1
5977695 0
6002627 1
6005992 0
6015690 0
6023477 1
6024951 1
6029523 0
6033608 0
6033737 0
6053431 0
6054919 1
6060663 0
6063428 0
6063693 1
6087229 1
6088585 0
6090019 0
6093013 0
6102868 1
6116566 0
6118563 1
6120193 0
6138216 0
6159524 0
6167179 1
6169512 0
6169586 0
6180537 0
6198699 0
6204273 0
6206509 0
