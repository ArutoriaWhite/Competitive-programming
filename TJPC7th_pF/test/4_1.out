2711551 6157683
